liquibase-standard/src/main/java/liquibase/command/core/HistoryCommandStep.java

```diff
@@ -37,6 +40,13 @@ public class HistoryCommandStep extends AbstractCommandStep {
                 .defaultValue(DateFormat.getDateTimeInstance(DateFormat.SHORT, DateFormat.SHORT), "Platform specific 'short' format")
                 .hidden()
                 .build();
+        TAGS_ARG = builder.argument("tags", Boolean.class)
+                .description("Include only tagged changesets")
+                .defaultValue(false)
+                .build();
+        TAG_ARG = builder.argument("tag", String.class)
```

Comment:
One other question, considering this is being used as a filter. Should it be a single value or could it be comma separate value and contain multiple tags that we should filter by?